srcpkgs/rpi-kernel/template

```diff
@@ -206,10 +206,6 @@ do_install() {
 	mkdir -p ${hdrdest}/net/mac80211/
 	cp net/mac80211/*.h ${hdrdest}/net/mac80211
 
-	# add dvb headers for external modules
```

Comment:
align with `srcpkgs/linux5.15/template`. align with mainline template on other occasions is wise too.

Reply:
wdym?

Reply:
work of many board kernels is duplicated work of the mainline template, so taking inspiration from the corresponding mainline template is nice , avoids hard-to-read diffs later on too